pkgs/development/compilers/gcc/5/default.nix

```diff
@@ -365,6 +367,7 @@ stdenv.mkDerivation ({
     ${if cross != null then crossConfigureFlags else ""}
     ${if !bootstrap then "--disable-bootstrap" else ""}
     ${if cross == null then platformFlags else ""}
+    ${if langJit then "--enable-host-shared" else ""}
```

Comment:
Can you verify that this patch doesn't change the hash for `pkgs.gcc` without langJit enabled? I'm worried that this line in particular will add a newline even without the option.